### PR TITLE
docs: update VS Code settings

### PR DESCRIPTION
**Pull request by shinGangan**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The 'Code Actions on Save and Auto' specification was changed in the 2023.11 VSCode update.
  
  - [November 2023 (version 1.85)](https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto)

The changes are quoted below.

> The setting value updates are as follows, with the previous boolean values to be deprecated in favor of the string equivalent.
>
> The options are:
>
> - explicit - Triggers Code Actions when explicitly saved. Same as true.
> - always - Triggers Code Actions when explicitly saved and on Auto Saves from window or focus changes.
> - never - Never triggers Code Actions on save. Same as false.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by shinGangan** [comment]
This PR is a low priority due to document updates.
Please review when you are available. 🙏

File: docs/5.community/4.contribution.md [hunk 1]
```diff
@@ -139,15 +139,15 @@ Under projects with configuration as shown below, Corepack will install `v7.5.0`
 
 We use [ESLint](https://eslint.org) for both linting and formatting with [`@nuxt/eslint-config`](https://github.com/nuxt/eslint-config).
 
-##### IDE Setup
+#### IDE Setup
```
Comment (DamianGlowala) [inline docs/5.community/4.contribution.md:142]: I believe this falls under the [Use ESLint](https://nuxt.com/docs/community/contribution#use-eslint) section, hence 5 levels of nesting.

Comment (shinGangan) [reply docs/5.community/4.contribution.md:142]: 5 levels doesn't seem to be able to jump, is this a problem on the Nuxt Content side?

> I believe this falls under the Use ESLint

Part of the ESLint section would be desirable. I agree with you on same this.

Comment (shinGangan) [reply docs/5.community/4.contribution.md]: fix this commit: 5d6c1231c8e9fe965504687b4931574ab51ff15f

**Review by DamianGlowala — APPROVED** [review]

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you!

**Pull request merged**